src/localization/locales/en.json

```diff
@@ -441,7 +441,8 @@
             "clearEvents": "Clear events",
             "showSystemProperties": "Show system properties",
             "start": "Start",
-            "stop": "Stop"
+            "stop": "Stop",
+            "fetch": "Fetch"
```

Comment:
"Fetch" seems a bit off-putting here. Would "Get events" or "Retrieve events" be better?